### PR TITLE
Added loss function and unit test for Poisson Loss

### LINKED ISSUE #9: Add more loss functions like cross-entropy...
Right we have implemented only Mean Squared Error and Logarithmic Error as our loss functions in our module. As we are going to implement more and more machine learning algorithms we will need more loss functions. There in this issue you can add one or more loss functions like cross-entropy etc.

### PR DESCRIPTION
**Pull request by yorozuya-2003**
#### References to other Issues or PRs or Relevant literature

Fixes #9

#### Brief description of what is fixed or changed
- Added class for Poisson Loss in **\MLlib\loss_func.py** 
- Added unit test corresponding to the same in **\MLlib\tests\test_loss_func.py** 

#### Other comments
The unit test approximately compares the output of the implemented loss function with that of the corresponding loss function already there in TensorFlow when using the same test example as input.

### CODE REVIEW

**Comment by rohansingh9001** [comment]
The loss method and the unit test looks good to me however it will not really be useful if you did not add a derivative method like the other loss classes in this file. Without a derivative function, the framework's models are unable to utilize loss classes.

**Review by rohansingh9001 — CHANGES_REQUESTED** [review]
The derivative function is missing from Poison loss class.

**Comment by yorozuya-2003** [comment]
#### References to other Issues or PRs or Relevant literature

Fixes #9

#### Brief description of what is fixed or changed
- Added derivative method for PoissonLoss class in **\MLlib\loss_func.py**

#### Other comments
I have tried adding the derivative method for the Poison loss class. Please suggest any corrections required. Thank you.
